docs/common-patterns.rst

```diff
@@ -151,8 +151,8 @@ restrictions highly readable.
             _;
         }
 
-        /// Make `_newOwner` the new owner of this
-        /// contract.
```

Comment:
These were using natspec by mistake.

Reply:
Why do you think so? It even uses the templating mechanism.

Reply:
Hm, this one may look like "a notice", but none of the other comments in this file look like one.

This also just reinforces my feeling that automatically taking those comments as notice is such a bad idea -- it is just not intuitive.

Reply:
I disagree. The wording sounds exactly like a "notice". Furthermore, there are only three public functions in this contract and the third does not have any comment at all.

Reply:
The wording of this one, but none of the others sound like so.

Reply:
I can remove this, but out of the entire documentation, it is this single contract having these two lines in the "implicit notice" format. Not sure it helps anyone, as it is not explained and there's no link to natspec.

Reply:
Some more discussions can be found here: https://gitter.im/ethereum/solidity-dev?at=60637bb4679f465566b058c9

Change removed from this PR.